src/main/scala/util/DescribedSRAM.scala

```diff
@@ -34,11 +34,12 @@ object DescribedSRAM {
     val uid = 0
 
     val omSRAM = DiplomaticObjectModelAddressing.makeOMSRAM(
-      desc = "mem-" + uid,
+      desc = desc,
       width = data.getWidth,
       depth = size,
       granWidth = granWidth,
-      uid = uid
+      uid = uid,
+      rtlModule = OMRTLModule(name)
```

Comment:
Can you pass arguments by name to the constructor of these OM case classes? There's no real rhyme or reason to the ordering of arguments, so passing arguments positionally is less easy to read than passing arguments by name.

```suggestion
      rtlModule = OMRTLModule(moduleName = name)
```